modules/compute/vm-instance/main.tf

```diff
@@ -261,6 +261,17 @@ resource "google_compute_instance" "compute_vm" {
     }
   }
 
+  dynamic "reservation_affinity" {
+    for_each = var.reservation_name == "" ? [] : [1]
+    content {
+      type = "SPECIFIC_RESERVATION"
+      specific_reservation {
+        key    = "compute.googleapis.com/reservation-name"
+        values = [var.reservation_name]
```

Comment:
Just curious - does the API (or underlying Terraform resource) actually take an array as input?  If so, shouldn't we change our interface to expose it as an array?

Reply:
It does take an array, but given that it's a specific array it only takes one value.  I think the idea is that `values` is a standardized input for key-value pairs, but in this case there's only going to be one value.